core/src/main/java/com/linecorp/armeria/common/util/BlockingTaskExecutorBuilder.java

```diff
@@ -136,6 +136,9 @@ public BlockingTaskExecutor build() {
         final ScheduledThreadPoolExecutor scheduledThreadPoolExecutor = new ScheduledThreadPoolExecutor(
                 numThreads, threadFactory);
         scheduledThreadPoolExecutor.setKeepAliveTime(keepAliveTimeMillis, TimeUnit.MILLISECONDS);
+        if (keepAliveTimeMillis > 0) {
```

Comment:
This will always be true as the setter doesn't allow 0 to be its value.

Why don't we set 0 as the default for `keepAliveTimeMillis`?